docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolhttprequest.md

```diff
@@ -1,13 +1,13 @@
 ---
 #https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
-title: HTTP Request Tool
-description: Documentation for the HTTP Request Tool node in n8n, a workflow automation platform. Includes details of operations and configuration, and links to examples and credentials information.
+title: HTTP Request Tool documentation
```

Comment:
Is there a reason why you didn't include 'node' here?

Reply:
Just a brain hiccup, I suspect, since I included it everywhere else. Updated!